<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSettings</class>
 <widget class="QWidget" name="FileSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="BuddyLabel" name="storeCoversInMpdDirLabel">
       <property name="text">
        <string>Save downloaded covers in music folder:</string>
       </property>
       <property name="buddy">
        <cstring>storeCoversInMpdDir</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="OnOffButton" name="storeCoversInMpdDir"/>
     </item>
     <item row="1" column="0">
      <widget class="BuddyLabel" name="label_4">
       <property name="text">
        <string>Save downloaded lyrics in music folder:</string>
       </property>
       <property name="buddy">
        <cstring>storeLyricsInMpdDir</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="OnOffButton" name="storeLyricsInMpdDir">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Save list of streams in music folder:</string>
       </property>
       <property name="buddy">
        <cstring>storeStreamsInMpdDir</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="OnOffButton" name="storeStreamsInMpdDir">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;i&gt;&lt;b&gt;NOTE:&lt;/b&gt; If you enable one of the above options, amd Cantata cannot access the music folder, then the files will be written as if the option was disabled. i.e. covers and lyrics will be stored in your cache folder, and streams will be saved in your config folder.&lt;/i&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BuddyLabel</class>
   <extends>QLabel</extends>
   <header>buddylabel.h</header>
  </customwidget>
  <customwidget>
   <class>OnOffButton</class>
   <extends>QCheckBox</extends>
   <header>onoffbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
